pkg/image/registry/imagesignature/strategy_test.go

```diff
@@ -0,0 +1,91 @@
+package imagesignature
+
+import (
+	"math/rand"
+	"reflect"
+	"testing"
+
+	"github.com/google/gofuzz"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	apitesting "k8s.io/kubernetes/pkg/api/testing"
+
+	"github.com/openshift/origin/pkg/api/v1"
+	"github.com/openshift/origin/pkg/image/api"
+)
+
+func fuzzImageSignature(t *testing.T, signature *api.ImageSignature, seed int64) *api.ImageSignature {
+	f := apitesting.FuzzerFor(t, v1.SchemeGroupVersion, rand.NewSource(seed))
+	f.Funcs(
+		func(j *api.ImageSignature, c fuzz.Continue) {
+			c.FuzzNoCustom(j)
+			j.Annotations = make(map[string]string)
+			j.Labels = make(map[string]string)
+			j.Conditions = []api.SignatureCondition{}
+			j.SignedClaims = make(map[string]string)
+
+			j.Content = []byte(c.RandString())
+			for i := 0; i < c.Rand.Intn(3)+2; i++ {
+				j.Labels[c.RandString()] = c.RandString()
```

Comment:
> why clear these just to re-fuzz them?

I'm not sure, how the fuzzer work. But nearly every time I run it without setting the labels myself, there were no labels set.